### DISCUSSION TITLE
Singularity or Docker support

### DISCUSSION THREAD

**Discussion by Chuang1118** [JinmiaoChenLab/DeepST#2] [Q&A]
Hello DeepST,

Thank you for your great work! Amazing benchmark.
I plan on using DeepST to integrate a multiple tissue sections(vertical integration).
1/ Can you provide singularity image v3 or dockerfile for reproducibility?
2/ In the part of Deconvolution, Cell2loc using q05 cell abundance, how about DeepST?

Greetings,
Chuang

**Comment by longyahui** [comment]
Hi Chuang,

     Thank you very much for your interest in our work.

     1) It's a good idea. We would follow your idea to provide dockerfile as soon as possible.
     2) For the deconvolution, DeepST retains 15% cell abundance by default. Users can change the parameter.

     Please let me know if you have any other questions. Thank you again.

     Greetings,
     Yahui
